la_palma_overview/__init__.py

```diff
@@ -215,74 +265,70 @@ def save_image(output_path, overview_config=None, verbose=False):
                 'http://fact-project.org/cam/skycam.php',
                 'http://www.gtc.iac.es/multimedia/netcam/camaraAllSky.jpg',
                 'http://www.magic.iac.es/site/weather/AllSkyCurrentImage.JPG',
-                'http://www.magic.iac.es/site/weather/can.jpg', 
+                'http://www.magic.iac.es/site/weather/can.jpg',
                 'http://www.fact-project.org/cam/cam.php',
                 'http://www.fact-project.org/cam/lidcam.php',
                 'http://iris.not.iac.es/axis-cgi/jpg/image.cgi',
                 'http://www.tng.iac.es/webcam/get.html?resolution=640x480&compression=30&clock=1&date=1&dummy=1456393525188',
-                #'http://www.gtc.iac.es/multimedia/netcam/camaraExt.jpg',
+                # 'http://www.gtc.iac.es/multimedia/netcam/camaraExt.jpg',
                 'http://www.magic.iac.es/site/weather/lastHUM6t.jpg',
                 'http://www.magic.iac.es/site/weather/lastWPK6t.jpg'
             ]
         }
 
-    #-----------------------------------
-    # the single image tiles of the stacked image are collected here
-    imgs = []
-
-    #-----------------------------------
+    # -----------------------------------
     # Collect all the images listed in the urls
-    for url in cfg['image_urls']:
-        try:
-            imgs.append(
-                download_and_resize_image_to_rows_and_cols(
-                    url, 
-                    cfg['img']['rows'], 
-                    cfg['img']['cols']
-                )
-            ) 
-        except:
-            imgs.append(empty_image(cfg['img']['rows'], cfg['img']['cols']))
-        Vprint(verbose, url)
-
-    #-----------------------------------
+    with ThreadPool(6) as pool:
```

Comment:
The two FACT images take 5 or more seconds each (because of these stupid waiting for the cameras).

I was quite annoyed by this during testing.
It's also not really parallel, as the GIL is still in effect (only python threads are used, not processes)